src/settings/components/AddAttributeDialog.jsx

```diff
@@ -54,7 +54,7 @@ const AddAttributeDialog = ({ open, onResult, definitions }) => {
             }
             return filtered;
           }}
-          options={options}
+          options={options.sort((a, b) => a.name.localeCompare(b.name))}
```

Comment:
Wouldn't it make more sense to do it at the definition? Also probably makes sense to memo it.